pkg/client/events/events.go

```diff
@@ -15,16 +15,17 @@
 package events
 
 import (
+	"time"
+
 	"github.com/IBM-Cloud/power-go-client/ibmpisession"
 	"github.com/IBM-Cloud/power-go-client/power/client/p_cloud_events"
 	"github.com/IBM/go-sdk-core/v5/core"
 	"github.com/ppc64le-cloud/pvsadm/pkg"
-	"time"
 )
 
 type Client struct {
 	instanceID string
-	client     *p_cloud_events.Client
+	client     p_cloud_events.ClientService
```

Comment:
any reason of removing the pointer from here?

Reply:
They introduced ClientService interface to access p_cloud_events.Client 's methods. Pointer to interface is not required I guess.